src/main/java/ch/njol/skript/expressions/ExprBiome.java

```diff
@@ -74,6 +74,7 @@ public Biome convert(final Location l) {
 	@Override
 	@Nullable
 	public Class<?>[] acceptChange(final ChangeMode mode) {
+		if(mode == ChangeMode.TOGGLE) return null;

```

Comment:
the return should be on the next line.
As well there should be a space between if and the parenthesis